op-e2e/bridge_test.go

```diff
@@ -110,129 +110,3 @@ func TestERC20BridgeDeposits(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, l2Balance, big.NewInt(100))
 }
-
-// TestBridgeGovernanceToken tests the L1StandardBridge bridge GovernanceToken functionality.
-func TestBridgeGovernanceToken(t *testing.T) {
```

Comment:
May I ask why you deleted this test?

Reply:
Since L2 governance token contract is converted not to be a predeploy contract, if we send the transaction to deploy it on L2 in e2e setup, some tests are broken. I thought it's unnecessary to modify all those tests to add this test, so I removed it. But if we change the default value of `enableGovernance` to false and turn it on only for the tests which need it, the issues are resolved. So I'll change it in this way.